README.md

```diff
@@ -111,6 +111,14 @@ Got a list of default packages you want installed every time you install a new N
 set --universal nvm_default_packages yarn np
 ```
 
+## `$nvm_data`
+
+If you want to set a custom directory for storing nvm data:
```

Comment:
Here's another way to put it. 


Set where nvm stores Node binaries and related data. Defaults to `$XDG_DATA_HOME/nvm` (~/.local/share/nvm) if unset.

```fish
set --global nvm_data ~/.nvm
```